webpack.config.environment.js

```diff
@@ -93,6 +94,11 @@ const BUNDLE = {
     localeDir: 'src/Locale',
     beditaPluginsRoot: 'plugins',
 
+    // alternate folders, cake4 pattern
+    alternateJsRoot: 'templates/Layout/js',
```

Comment:
Template layout folder in Cake 4 is `templates/layout` 

Javascript files could be set somewhere else, in many projects we use `resources/js` for instance as js root and use `templates` for `.twig` files only

Reply:
good point, but not working with existing plugins that use `templates/Layout/js`. The main idea of this PR was to maintain retrocompatibility (fix bug for those plugins introduced by manager passage to cake4) plus moving forward a step towards cake 4 folder layout